<launch>
    <param name="/use_sim_time" value="true" />
    <node name="WholeBodyValkyrieControl" pkg="valkyrie_rqt_reconfigure" type="WholeBodyValkyrieControl" output="screen" />
</launch>
